static/app/routes.tsx

```diff
@@ -550,6 +550,10 @@ function buildRoutes() {
             () => import('sentry/views/settings/projectMetrics/projectMetricsDetails')
           )}
         />
+        <Route
+          path=":spanAttribute/edit/"
+          component={make(() => import('sentry/views/settings/projectMetrics'))}
+        />
```

Comment:
Just wondering here: Would there be any benefit in using a query param instead? 🤔

Reply:
I was wondering the same thing, but since we have everything else we need on the page, for now we don't need the other params